migrations/20190212004845-tags-rlinks-not-null-multiple-models.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+const db = require('../db/index');
+const constants = require('../db/constants');
+
+const modelsAndCols = {
+  Aspect: ['tags', 'relatedLinks'],
+  Generator: ['tags'],
+  GeneratorTemplate: ['tags'],
+  Perspective:
+    ['aspectFilter', 'aspectTagFilter', 'subjectTagFilter', 'statusFilter'],
+  Sample: ['relatedLinks'],
+};
+
+function updateRowsAndColumns(qi, seq, modelName, colNamesArr) {
+  const whereArr = [];
+  colNamesArr.forEach((cname) => {
+    const o = {};
+    o[cname] = { $eq: null };
+    whereArr.push(o);
+  });
+
+  const model = db[modelName];
+  return model.findAll({ // find records with any of given columns = null
+    where: { $or: whereArr },
+  })
+    .then((records) => Promise.all(
```

Comment:
This might run into the same problem I did in the owner migration. You should use Promise.mapSeries to make sure it runs the queries one-at-a-time.